src/annoylib.h

```diff
@@ -1158,11 +1158,21 @@ template<typename S, typename T, typename Distance, typename Random>
     }
   }
 
-  inline Node* _get(const S i) const {
+  Node* _get(const S i) const {
```

Comment:
There's no real code-change but more code-style as any header function is forced to be inlined by the compiler no matter whether it's marked `inline` or not (it would break ORD otherwise).